### ISSUE TITLE
Judgement of Wisdom & Consecration

### ISSUE THREAD

**Issue by ccshiro** [ccshiro/cc-buglist#15]
Report: "Paladin ability Judgement of Wisdom and Consecration function - Judgement of Wisdom should be able to proc on the first damage tick of Consecration ability (only the first, rest do not have chance to proc it), currently doesn't at all. (http://maintankadin.failsafedesign.com/forum/viewtopic.php?f=6&t=6203)"

Question: The provided source seems a bit weak for a report that goes against our current understanding of procs. We're looking for a more reliable source; this includes videos and/or a trustworthy textual source (such as elitistjerks).

**Comment by Skibbli** [comment]
The best video I have been able to find so far is http://www.youtube.com/watch?v=ZBT4HqrdCpY - it shows a Gruul fire mage throughout the fight, and you can see the combat lag for when JoW procs. It only procs when the spells land, not when the fireball or ignite dots tick, which would suggest that consecrate should only apply on initial application. I will try and find some more evidence of it.

EDIT - http://www.youtube.com/watch?v=cdP7WhkKA8Q - Shadow Priest, also on Gruul. During Shatter (no casts) no procs from dots.

**Comment by AtheneGenesis** [comment]
Usefull informations about this:

http://maintankadin.failsafedesign.com/forum/viewtopic.php?f=6&t=6812

"The only thing I've ever, ever noticed the first tick of consecrate being useful for is breaking rogues out of stealth. The initial "blast" has knocked me out of stealth on my rogue before (I believe). It seems to act like a damaging spell (arcane explosion) yet the initial "blast" does zero damage."

The "blast" (the initial application of the consecration) count as a mage arcane explosion but with 0 damage, the first tick arriving 1 second later, this blast if not resisted may proc JoW (not JoL) and can be use to break stealth.

Edit: Found some old wowcombatlog back from 2.4.0, You can see a JoW proc after a consecration and before any white hit:
![jow consec](https://cloud.githubusercontent.com/assets/6951059/4917782/7129df4c-64e9-11e4-8027-a36e41945c10.PNG)

**Comment by ccshiro** [comment]
@Skibbli That's indeed how it works for us as well.

@AtheneGenesis That is very interesting. The explanation makes sense as well.

I will leave this open a bit more in case further evidence is found.

**Comment by Barroth** [comment]
may be possible that it is sth a lot of aoe spells have, found a patchnote similar to the strealthbreaking:

"Force of Nature: This ability will no longer automatically break 
   stealth on victims in its casting area.  Nearby stealthed characters 
   may still be attacked if they are too close to the summoned Treants."

I feel like it could be related to the spell targets(there is aoe spells with TARGET_INSTANT and some without INSTANT)

**Comment by AtheneGenesis** [comment]
@ccshiro If you want, I have 2 old wowcombatlog.txt, one that was used by the recount author to make it compatible with 2.4.0 combat log and another I found on the internet. Both are from 2008 and may contain useful informations ;)

Where can I PM you a link for the DL if you want it ?

**Comment by Barroth** [comment]
you can pm shiro any time you want at  irc: irc.rizon.net (defaultport 6667)

**Comment by ccshiro** [comment]
@AtheneGenesis thanks, I received your PM.

**Comment by ccshiro** [comment]
Got PM'd this by Mustonen: https://www.youtube.com/watch?v=B6RIMIBsIPk

It's a HD paladin tank video from TBC where you can read the combat log.

**Comment by Barroth** [comment]
The combatlog only shows damage taken, heal/mana gain and buff fading/receiving. So its hard to sync it to the actions the paladin uses(especially with the spell delay)

Edit: Found sth else: http://www.wowwiki.com/Consecration?oldid=1390450#Notes "The First tick was resisted and it still did 8 ticks". Could be quite possible, that they mistook the "initial Blast" that got resisted for the first tick

Edit2: I've found a clue why it works with consecration: 
Vanilla patchnote:
 "Consecration - No longer displays a debuff icon on targets in the 
  area of effect.".

 Since debuff applications, like Dots, are able to trigger it, It probably is able to trigger the Judgement of Wisdom because it actually is a "debuff" which is just not shown anymore.
